goss/goss-windows.yaml

```
@@ -62,7 +62,7 @@ command:
     exit-status: 0
     stdout:
       - /16\.\d+\.\d+\.\d+/
-    skip: {{ not (eq .Env.AGENT_OS_VERSION "2019") }}
+    skip: {? {not (eq .Env.AGENT_OS_VERSION "2019"): ''} : ''}
```

```suggestion
    skip: {{ not (eq .Env.AGENT_OS_VERSION "2019") }}
```

Another manifest error cc @smerle33 (with ruby and python) leading to this change